src/pages/Registration/Signup/Form.tsx

```diff
@@ -23,7 +23,7 @@ export default function Form({ classes = "" }: { classes?: string }) {
   return (
     <form
       onSubmit={submit}
-      className={`${classes} padded-container w-full max-w-[37.5rem] grid`}
+      className={`${classes} justify-center gap-8 padded-container w-full max-w-[37.5rem] grid`}
     >
       <h3 className="text-3xl font-bold text-center">
```

Comment:
nit: `text-center` can probably be removed (as it's already justified to center).

Reply:
@misicnenad it didnt work cause justify-center center aligns the block but thetext within the block remains left aligned
text-center aligns the the text within the block of <h3> to center